packages/subgraph/tests/dao/dao_v1_0_0.test.ts

```diff
@@ -260,13 +258,17 @@ describe('handleNativeTokenDeposited', () => {
 describe('handleDeposited: ', () => {
   beforeAll(() => {
     daoTokenContract = new ExtendedERC20Contract().withDefaultValues();
-    let totalSupply = '10';
-    daoTokenContract.mockCall_createTokenCalls(totalSupply);
+
+    createERC20TokenCalls(
+      DAO_TOKEN_ADDRESS,
+      ERC20_AMOUNT_FULL,
+      'DAO Token',
+      'DAOT',
+      '6'
```

Comment:
1- I see you used `createERC20TokenCalls` in the extended classes,
this mean u can used `mockCall_createTokenCalls(totalSupply)` normally and you do not need to refactor here

2- why `createERC20TokenCalls`  do not return total supply as well, 
perhaps we need some refactoring in the common